web/react/stores/browser_store.jsx

```diff
@@ -25,9 +22,9 @@ class BrowserStoreClass {
         this.isLocalStorageSupported = this.isLocalStorageSupported.bind(this);
 
         var currentVersion = localStorage.getItem('local_storage_version');
-        if (currentVersion !== BROWSER_STORE_VERSION) {
+        if (currentVersion !== global.window.config.Version) {
```

Comment:
yea,  This is more a bug in how browser_store wipes its storage on push.  I'll file a fix to skip removal of edit msg history.